subprojects/packagefiles/flac/meson.build

```diff
@@ -1,8 +1,9 @@
-project('flac', 'c', version: '1.3.3')
+project('flac', 'c', version: '1.3.4')
 
 cc = meson.get_compiler('c')
 
 add_project_arguments('-DHAVE_CONFIG_H', language: 'c')
+add_project_arguments('-D_DARWIN_C_SOURCE', language: 'c')
```

Comment:
This should probably be guarded by `if host_machine.system() == 'darwin'`-

Reply:
Done. Pointless IMO.